tools/projmgr/src/ProjMgr.cpp

```diff
@@ -99,12 +100,13 @@ int ProjMgr::RunProjMgr(int argc, char **argv) {
   cxxopts::Option load("l,load", "Set policy for packs loading [latest|all|required]", cxxopts::value<string>());
   cxxopts::Option missing( "m,missing", "List only required packs that are missing in the pack repository", cxxopts::value<bool>()->default_value("false"));
   cxxopts::Option schemaCheck( "n,no-check-schema", "Skip schema check", cxxopts::value<bool>()->default_value("false"));
+  cxxopts::Option noUpdateRte( "no-update-rte", "Skip creation of RTE directory and files", cxxopts::value<bool>()->default_value("false"));
```

Comment:
Since all flags have short and long options, for consistency should be accepted a short option here as well?